tests/runtime/call

```diff
@@ -216,12 +216,12 @@ RUN bpftrace -v -e 'BEGIN { @[nsecs] = strftime("%m/%d/%y", nsecs); exit();}'
 EXPECT @\[[0-9]*\]: [0-9]{2}\/[0-9]{2}\/[0-9]{2}
 TIMEOUT 5
 
-NAME print_avg_map_top
-RUN bpftrace -e 'i:ms:1 { @[nsecs/100000] = avg(1) } i:ms:10 { print(@, 3, 0); exit(); } END {clear(@)}' | grep -c -v "^ *$"
-EXPECT ^4$
+NAME print_avg_map_args
+RUN bpftrace -e 'BEGIN { @["a"] = avg(10); @["b"] = avg(20); @["c"] = avg(30); @["d"] = avg(40); print(@, 2, 10); print("END"); clear(@); exit(); }'
+EXPECT Attaching 1 probe\.\.\.\n@\[c\]: 3\n@\[d\]: 4\n\nEND
```

Comment:
I think the regex does a line-by-line match which means I think you can do this
```suggestion
EXPECT @\[c\]: 3\n@\[d\]: 4\n\nEND
```

Reply:
Hmm actually, I do this on purpose. I think the 'EXPECT' actually means 'CONTAINS' in runtime test framework, in other words `EXPECT @\[c\]: 3\n@\[d\]: 4\n\nEND` would allow something like:
```
Attaching 1 probe...
@[a]: 1
@[b]: 2
@[c]: 3
@[d]: 4

END
```
because it contains specified pattern. Prefix the pattern with the line 'Attaching 1 probe...' can prevent it from ignoring the wrong output.

Reply:
Hmm, what "wrong output" would it avoid? Isn't it enough to match only
```
@[a]: 1
@[b]: 2
@[c]: 3
@[d]: 4

END
```
Why would we need to match the entirety of the output?

Reply:
I'm not sure if I got it right here, but here is what I think. Since we have `top = 2`, the script below should only print the top 2 entries:
```
# bpftrace -e 'BEGIN { @["a"] = avg(10); @["b"] = avg(20); @["c"] = avg(30); @["d"] = avg(40); print(@, 2, 10); print("END"); clear(@); exit(); }'

Attaching 1 probe...
@[c]: 3
@[d]: 4

END
```
And since we want `top` to be effective, we do not actually want to see all four entries being printed out. If the output contains four entries, `top` is not functioning properly; but in this case, the four-entry output can still pass the test with `EXPECT @\[c\]: 3\n@\[d\]: 4\n\nEND` because the output contains this pattern. 

If I'm not getting it wrong, the output with all four entries should be the wrong output we may want to avoid here. The expected output should contains exactly `@[c]: 3` and `@[d]: 4`.

Reply:
Ah yeah that makes sense. Thanks for the explanation